tests/base_test.py

```diff
@@ -118,7 +119,11 @@ def setUpClass(cls):
             oauth_client_secret=OAUTH_CLIENT_SECRET,
             jwt_algorithm=test_settings.JWT_ALGORITHM,
         )
-        cursor.execute(sql)
+
+        try:
+            cursor.execute(sql)
+        except psycopg2.errors.UniqueViolation:  # entry already in database
+            pass
```

Comment:
Hopefully this won't be a problem after #10711 but I can recheck it

Reply:
I looked into this a little further. This is still a problem after #10711. It's related to the TODO statements I added showing where we have some package loads in test case setup that _depend_ on running _outside_ of a transaction, which isn't ideal:

https://github.com/archesproject/arches/blob/350e4a7cd2cca3b17176cf31bbcbf5b5282ff695/tests/views/api_tests.py#L41-L44

I'll open an issue and ref them in the TODO statements. If we fix that, I expect we wouldn't need this workaround.

However, I don't think this workaround is doing what it appears. When I revert the bugfix that causes the api test to fail, and I run `python3 manage.py test tests.views.api_tests tests.views.auth_tests --settings=tests.test_settings`, this except isn't reached because Django reraises `UniqueViolation` as `IntegrityError`. However, if I change this to catch IntegrityError, then it causes far more failures because the transaction is aborted and doesn't clean up after itself and all subsequent tests fail.

I'm feeling like we should just remove the workaround and wait for the follow-up issue to be fixed. WDYT?

Reply:
Hmm let's keep this fix for now but you should request changes on this PR so it can't get merged in. That way the fix is still there in case the followup doesn't work or @apeters decides to merge it in knowing that the followup is not far behind.